composer.json

```diff
@@ -66,9 +66,6 @@
             "Tests/"
```

Comment:
This property is not used outside of optimizing the autoloader (which optimize the ps4 autoloading into a classmap).

This PR won't cause any bug except if someone want to run the tests with an optimized autoloader, but I'm not sure it's an use case that has to be supported.